judge-d-server/src/main/java/dev/hltech/dredd/interfaces/rest/contracts/ContractsController.java

```diff
@@ -46,17 +45,7 @@ public ServiceContractsDto create(@PathVariable(name = "provider") String provid
         ));
     }
 
-    @GetMapping(value = "/contracts", produces = MediaType.APPLICATION_JSON_VALUE)
-    @ApiOperation(value = "Get names of services with registered contracts", nickname = "get names of services")
-    @ApiResponses(value = {
-        @ApiResponse(code = 200, message = "Success", response = String.class, responseContainer = "list"),
-        @ApiResponse(code = 400, message = "Bad Request"),
-        @ApiResponse(code = 500, message = "Failure")})
-    public List<String> getAvailableServiceNames() {
-        return serviceContractsRepository.getServiceNames();
-    }
-
-    @GetMapping(value = "/contracts/{serviceName}", produces = MediaType.APPLICATION_JSON_VALUE)
+     @GetMapping(value = "/contracts/{serviceName}", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
If you moved GET "/services", this should be moved as well.

Reply:
After our discussion we postpone moving this endpoint to separate controller nor change its path until we have a clear vision what we are trying to achieve.